src/modules/UI/scenes/SendConfirmation/action.js

```diff
@@ -28,6 +39,55 @@ export const updateAmount = (nativeAmount: string, exchangeAmount: string, fiatP
   dispatch(createTX({ nativeAmount, metadata }, false))
 }
 
+type EdgePaymentProtocolUri = EdgeParsedUri & { paymentProtocolURL: string }
+
+export const paymentProtocolUriReceived = ({ paymentProtocolURL }: EdgePaymentProtocolUri) => (dispatch: Dispatch, getState: GetState) => {
+  const state = getState()
+  const walletId = getSelectedWalletId(state)
+  const edgeWallet = getWallet(state, walletId)
+
+  Promise.resolve(paymentProtocolURL)
+    .then(paymentProtocolURL => {
```

Comment:
A tiny styling comment, i think it's prettier like this:
then(paymentProtocolURL => getPaymentProtocolInfo(edgeWallet, paymentProtocolURL))

Reply:
I like to line up all the `.then`'s

Reply:
i prefer to line up all the `.then`'s